core/indigo-core/reaction/src/reaction_json_saver.cpp

```diff
@@ -311,7 +310,7 @@ void ReactionJsonSaver::saveReaction(BaseReaction& rxn)
         merged = std::make_unique<Molecule>();
     }
 
-    if (rxn.metaData().size())
+    if (rxn.isMultistep())
```

Comment:
Not sure if this is correct. It's the only metadata for reaction?

Reply:
Metadata is saved in any case. For multistep reaction arrows and pluses are included as metadata and for a single reaction not included. To avoid confusion renamed saveReactionWithMetadata to saveMultistepReaction.